### ISSUE TITLE
[kickasstorrents-to] (testing) Test search in kickasstorrents.to => Found no results while trying to browse this tracker. This may be an issue with the indexer, or other indexer settings such as search freeleech only etc.

### ISSUE THREAD

**Issue by Dwarf8** [Jackett/Jackett#14870]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
11-28 08:37:55 Info Starting Jackett v0.21.1264
11-28 08:38:07 Info Environment version: 6.0.25 (/app/Jackett/)
11-28 08:38:07 Info OS version: Unix 5.16.0.0 (64bit OS) (64bit process)
11-28 08:38:07 Info Jackett variant: CoreLinuxMuslAmdx64
11-28 08:38:07 Info File /etc/issue: Welcome to Alpine Linux 3.17
11-28 08:38:07 Info Running in Docker: Yes (image build: v0.20.3583-ls96)
11-28 08:38:07 Info ThreadPool MaxThreads: 32767 workerThreads, 1000 completionPortThreads
11-28 08:38:07 Info App config/log directory: /config/Jackett
11-28 08:38:07 Info Using proxy: Disabled
11-28 08:38:07 Info Using FlareSolverr: No
11-28 08:38:07 Info Using HTTP Client: HttpWebClient2
11-28 08:38:07 Info Loading Native indexers ...
11-28 08:38:08 Info Loaded 73 Native indexers.
11-28 08:38:08 Info Loading Cardigann indexers from: /config/cardigann/definitions/, /etc/xdg/cardigan/definitions/, /app/Jackett/Definitions
11-28 08:38:14 Info Loaded 530 Cardigann indexers.
11-28 08:38:14 Info Loaded 603 indexers in total
11-28 08:38:14 Info Adding aggregate indexer ('all' indexer) ...
11-28 08:38:14 Info Adding filter indexer ('type:public' indexer) ...
11-28 08:38:14 Info Adding filter indexer ('type:private' indexer) ...
11-28 08:38:14 Info Adding filter indexer ('type:semi-public' indexer) ...
11-28 08:38:15 Info Jackett startup finished in 34.914 s
Hosting environment: Production
Content root path: /app/Jackett/Content
Now listening on: http://[::]:9117
Application started. Press Ctrl+C to shut down.
[ls.io-init] done.
```

### Description

Kickasstorrents can't found anything.

### Logged Error Messages

```text
11-28 08:38:42 Info Torznab search in kickasstorrents.to => Found 0 releases [13821ms]
```

**Comment by garfield69** [comment]
I get results for all 4 domains of the kickasstorrents-to indexer, which suggests the problem is at your end.
Please enable the **enhanced** logging and repeat your test and upload the full log so we have something useful to look at.
See https://github.com/Jackett/Jackett/wiki/Troubleshooting/#how-to-provide-a-jackett-enhanced-log-for-a-ticket

**Comment by Dwarf8** [comment]
This log?

[Jackett_logs.txt](https://github.com/Jackett/Jackett/files/13485244/Jackett_logs.txt)

**Comment by ilike2burnthing** [comment]
You're getting a mix of `403 Forbidden` and `Access denied` errors, which suggests your IP is being purposefully blocked.

Are you using a VPN, proxy, or VPS/seedbox?

**Comment by Dwarf8** [comment]
I use a NUC with OpenMediaVault and Docker, with "nginx proxy manager" to have radarr.mydomain.com, sonarr.mydomain.com, jackett.mydomain.com, etc, but i don't think thats the problem, it was working previusly...

**Comment by ilike2burnthing** [comment]
I'm not asking about a reverse proxy, I'm asking about a traffic routing proxy or VPN, one that changes how your public IP is seen by sites you're accessing.

If you're not using them, but you're using automation software like Sonarr et al, then you could disable or remove Kickass in them, and remove it from Jackett. Then after a day or two you could add it again, to see if it's working.

Alternatively, you could us a VPN or proxy and try a few different servers to see if that bypasses the block.

**Comment by Dwarf8** [comment]
Ok, i'm not using proxy or VPN, so i will remove Kickass and try again next week...

Thanxs!

**Comment by garfield69** [comment]
The logs point to cloudflare rate limiting your access by banning you temporarily from accessing the  website.
This suggests that your Sonarr and/or Radarr are set up with search query per period cycle that are too high.
Or that you have not enabled the Jackett cache or that this is set to flush too soon,
As ilike2burnthing pointed out, you need to have a break from using the web site  in order to reset the temporary ban.

**Comment by garfield69** [comment]
Sonarr/Settings/Indexers/RSS Sync Interval
Radarr/Settings/Indexers/RSS Sync Interval

**Comment by Dwarf8** [comment]
Sonarr -> 15 minutes.
Radarr ->120 minutes.

What is the recommended value?

**Comment by ilike2burnthing** [comment]
The defaults are fine, Jackett's default cache timeout means Sonarr will perform 1 actual search followed by 2 cache searches, so you're only requesting from Kickass and other sites every 45mins.

**Comment by Dwarf8** [comment]
Ok, I will try again next week!

Thanxs!

**Comment by al3x1337** [comment]
Im having the same issue as OP with EZTV. I use Adgaurd for DNS which I turned off and tried Jackett (I'm a Prowlarr user) but am getting the same error-- `Query successful, but no results were returned from your indexer. This may be an issue with the indexer, your indexer category settings, or other indexer settings such as search freeleech only etc.`

I have Flaresolverr which seemed to be required; once set up the Flaresolver errors in the logs went away. I also disabled Caching for this test.

The only Site Link out of the list that did not throw said message is https://eztv.unblockninja.com/ but running a query resolved in 0 results-- Manual search in E`ZTV for 90 day => Found 0 releases [3630ms]`

Im able to visit the sites manually and download from there, everything was working 1/2 weeks ago.

Any further insight would be great into this...
